tests/main/xdg-settings/task.yaml

```diff
@@ -69,6 +71,17 @@ execute: |
     touch /usr/bin/zenity
     mount --bind /bin/true /usr/bin/zenity
 
+    if is_cgroupv2 ; then
+        # we are expecting the test to fail on a cgroup v2 system
+        if ensure_xdg_settings_output "http://smoke-test-cgroupv2.com" 2> stderr.log ; then
```

Comment:
We should probably `rm -f stderr.log` in restore section.

Reply:
@stolowski, thanks for reviewing, it is automatically removed. The test directory is saved before the test execution and restored after the execution, so it is not needed to clean up the test directory.